### ISSUE TITLE
Display/Render toolbar at user specified position

### ISSUE THREAD

**Issue by qasid** [liferay/alloy-editor#638]
Currently toolbar render w.r.t. selection range and renders above , in middle or at end of selection. Is there a way we can render toolbar at any user specified position using configuration property 
{ e.g 

 toolbar-position: [ "top",  "left",  "right",  "bottom",  "middle" ],
 or 
 toolbar-position: [ top: 10 , left:  20],  (where 10 and 20 are user defined values) 

 }

**Comment by ipeychev** [comment]
No, there is no such feature.

**Comment by badcommandorfilename** [comment]
Hi qasid,

I was looking for a similar solution - in my case I was using Alloy Editor inside an iframe and the toolbar was being placed relative to the window root body instead of the inner iframe body.

I found issue #276 and I was able to solve it by using the [uiNode](http://alloyeditor.com/api/classes/Core.html#property_uiNode) property, which places all the menus/toolbars relative to the specified node. In my case, I used the iframe parent node and it re-positioned the toolbar correctly. In your case you might be able to move the menu position around with a floating div.

Hope that helps!
